### PR TITLE
[doc][ybm] Slow query latency histogram

### PR DESCRIPTION
**Pull request by ddhodge**
Slow query latency histogram

@netlify /preview/yugabyte-cloud/cloud-monitor/cloud-queries-slow/

### CODE REVIEW

File: docs/content/preview/yugabyte-cloud/cloud-monitor/cloud-queries-slow.md [hunk 1]
```diff
@@ -12,30 +12,48 @@ menu:
...
 | Column          | Description                                                  |
 | --------------- | ------------------------------------------------------------ |
+| Query           | The query statement.               |
 | User            | The name of role used to access YSQL database.               |
-| Total time      | The total duration (in milliseconds) of all iterations this query has taken. |
+| Database        | The YSQL database used by the query.                         |
+| Avg time        | The average execution time (in milliseconds) for this query. |
 | Count           | The total number of times this type of query has executed.   |
+| Total time      | The total duration (in milliseconds) of all iterations this query has taken. |
 | Rows            | The total number of database table rows returned across all iterations of this query |
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-cloud/cloud-monitor/cloud-queries-slow.md:46]: The total number of database table rows returned across all iterations of this query.

File: docs/content/preview/yugabyte-cloud/cloud-monitor/cloud-queries-slow.md [hunk 2]
```diff
@@ -12,30 +12,48 @@ menu:
...
-| Database        | The YSQL database used by the query.                         |
+| Min time        | The minimum execution time (in milliseconds) for this query. |
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-cloud/cloud-monitor/cloud-queries-slow.md:47]: I don't see Min time and Max time in the screenshot. Is it not displayed by default ?

**Review by daniel-yb — APPROVED** [review]

**Review by aishwarya24 — APPROVED** [review]
LGTM. For YBA, maybe copy it to older versions as well.

File: docs/content/preview/yugabyte-cloud/cloud-monitor/cloud-queries-slow.md (hunk 2 above)
Comment (ddhodge) [reply docs/content/preview/yugabyte-cloud/cloud-monitor/cloud-queries-slow.md:47]: no

**Pull request merged**